sycl/include/sycl/ext/oneapi/experimental/fixed_size_group.hpp

```diff
@@ -47,7 +47,7 @@ template <size_t PartitionSize, typename ParentGroup> class fixed_size_group {
 
   id_type get_group_id() const {
```

Comment:
`operator fragment<ParentGroup>() const;` is missing, but we don't have `fragment`  implementation yet, so let's just add a `TODO` comment as a reminder to add it.

Reply:
done